packages/validator/src/metrics.ts

```diff
@@ -325,6 +325,12 @@ export function getMetrics(register: MetricsRegister, gitData: LodestarGitData)
       help: `Current health status of the beacon(s) the validator is connected too. ${renderEnumNumeric(BeaconHealth)}`,
     }),
 
+    validatorStatuses: register.gauge<{status: string}>({
+      name: "vc_validator_status",
```

Comment:
could consider plural in the metric name as well, not sure tho

```suggestion
      name: "vc_validator_statuses",
```